tool/src/java/org/sakaiproject/gradebookng/tool/panels/GradeItemCellPanel.java

```diff
@@ -215,11 +216,13 @@ protected void onSubmit(final AjaxRequestTarget target) {
 					}
 
 					//refresh the components we need
-					send(getPage(), Broadcast.BREADTH, new ScoreChangedEvent(studentUuid, categoryId, target));
 					target.addChildren(getPage(), FeedbackPanel.class);
 					target.add(getParentCellFor(this));
 					target.add(this);
 
+					//trigger async event that score has been updated and now displayed
+					target.appendJavaScript(String.format("$('#%s').trigger('scoreupdated.sakai')", this.getMarkupId()));
+
```

Comment:
@steveswinsburg, I couldn't see a nice way to programatically trigger a callback, but I did find a Wicket example that did something similar to this.  Let me know if you know of any sweet Wicket ways to do this! :rocket: